blackbox/docs/sql/system.txt

```diff
@@ -937,6 +937,8 @@ There are currently four checks:
 +------------------+------------------------------------+
 | 5                | Number of partitions.              |
 +------------------+------------------------------------+
+| 6                | Java 7 deprecation warning.        |
```

Comment:
pls change the description above this table, it currently writes about 4 checks ;) (maybe just reword it to skip number of checks so we won't have to change it always)